types/bigint/index.d.ts

```diff
@@ -352,7 +352,7 @@ declare namespace BigInt {
     /**
      * do x=x+n where x is a bigInt and n is an integer.
      *
-     * @private Intend to be internal function.
```

Comment:
This one I'm wondering about... why is this documented publicly if it's meant to be an internal function? Should it be removed from the types?

My personal instinct as someone who has never used `@types/bigint` would be to lean towards removing it. cc @Evgenus - are you up for discussing here so we can keep it in the context of `jsdoc/check-tag-names`?

Reply:
Great! I'm adding in a `@note` to make this more in line with JSDoc, since it kind of is a "note" / not part of the description.